### DISCUSSION TITLE
Location issue when applying

### DISCUSSION THREAD

**Discussion by 18yz153** [community/community#144884] [GitHub Education]
### Hi everyone,

I think my campus is not recorded in github database. I am UIUC student in Chicago. 
Chicago Office
200 South Wacker Drive, 7th Floor
Chicago, IL 60606

**Comment by FeiSuoerDe** [comment]
After my attempt, University of Illinois Chicago exists in the database, please enter "University of Illinois Chicago" directly

**Reply by 18yz153** [reply]
I am UIUC student. I only have illinois.edu

**Reply by FeiSuoerDe** [reply]
My mistake, trying to type in "University of Illinois Urbana-Champaign"?

**Reply by FeiSuoerDe** [reply]
The database search will take some time, please be patient, even if I type in 'UIUC', after about five seconds of waiting, the drop-down box option for your school will appear

**Reply by FeiSuoerDe** [reply]
Or do you want to apply directly from the "University of Illinois"?

**Reply by FeiSuoerDe** [reply]
Use **atmos.uiuc.edu**？ This is the email address I found.
